### PR TITLE
FAQ and template update on migrating App Insights to Workspace-based

### PR DESCRIPTION
**Pull request by Aboli-msft**

### CODE REVIEW

File: 01-Setup and getting started/README.md [hunk 1]
```diff
@@ -3,7 +3,13 @@
...
 </br>
 
-> We have updated our auto update feature to use AAD Authentication for Function App deployment instead of basic auth or publishing profile. Please refer details [here](../FAQs/README.md/#6-what-are-the-recommended-authentication-settings-for-all-app-services-or-function-apps). 
+> We have updated our auto update feature to use AAD Authentication for Function App deployment instead of basic auth or publishing profile. Please refer details [here](../FAQs/README.md/#6-what-are-the-recommended-authentication-settings-for-all-app-services-or-function-apps).
+
+> FAQ for guidance on migrating classic Application Insights to Workspace-based [here](../FAQs/README.md/#7-how-can-i-migrate-classic-application-insights-to-workspace-based-if-i-migrate-application-insights-azsk-azts-appinsights-xxxxx-to-workspace-based-will-there-be-any-impact-to-azts-scan).
+
+</br>
+
+> We have
```
Comment (asimks) [inline 01-Setup and getting started/README.md:12]: What we intend to write here. I think we have missed completing this

File: 01-Setup and getting started/README.md [hunk 2]
```diff
@@ -3,7 +3,13 @@
 
 </br>
 
-> We have updated our auto update feature to use AAD Authentication for Function App deployment instead of basic auth or publishing profile. Please refer details [here](../FAQs/README.md/#6-what-are-the-recommended-authentication-settings-for-all-app-services-or-function-apps). 
+> We have updated our auto update feature to use AAD Authentication for Function App deployment instead of basic auth or publishing profile. Please refer details [here](../FAQs/README.md/#6-what-are-the-recommended-authentication-settings-for-all-app-services-or-function-apps).
+
+> FAQ for guidance on migrating classic Application Insights to Workspace-based [here](../FAQs/README.md/#7-how-can-i-migrate-classic-application-insights-to-workspace-based-if-i-migrate-application-insights-azsk-azts-appinsights-xxxxx-to-workspace-based-will-there-be-any-impact-to-azts-scan).
```
Comment (asimks) [inline 01-Setup and getting started/README.md:8]: better to say Workspace-based resource

File: FAQs/README.md [hunk 3]
```diff
@@ -12,6 +12,8 @@
  4. [While installing AzTS solution I have provided my preferences for telemetry collection i.e. anonymous AzTS usage data and organization/team contact details. How do I update my preferences now?](#4-while-installing-azts-solution-i-have-provided-my-preferences-for-telemetry-collection-ie-anonymous-azts-usage-data-and-organizationteam-contact-details-how-do-i-update-my-preferences-now)
 5. [On running the Autoupdater function I am encountering an error related to conflicts in function runtime version.](#5-on-running-the-autoupdater-function-i-am-encountering-an-error-related-to-conflicts-in-function-runtime-version)
 6. [What are the recommended authentication settings for all App services or Function Apps?](#6-what-are-the-recommended-authentication-settings-for-all-app-services-or-function-apps)
+7. [How can I migrate classic Application Insights to Workspace-based? If I migrate Application Insights AzSK-AzTS-AppInsights-xxxxx to Workspace-based, will there be any impact to AzTS scan?](#7-how-can-i-migrate-classic-application-insights-to-workspace-based-if-i-migrate-application-insights-azsk-azts-appinsights-xxxxx-to-workspace-based-will-there-be-any-impact-to-azts-scan)
```
Comment (asimks) [inline FAQs/README.md:15]: same as above, it seems we ended the statement in mid way. Workspace-based resource seems better.

File: FAQs/README.md [hunk 4]
```diff
@@ -114,6 +116,25 @@ Using the native enterprise directory for authentication ensures that there is a
 <br>
 <br>
 
+### **7. How can I migrate classic Application Insights to Workspace-based? If I migrate Application Insights AzSK-AzTS-AppInsights-xxxxx to Workspace-based, will there be any impact to AzTS scan?**
+It is recommended to migrate to classic Application Insights to Workspace-based before retirement date(29th February, 2024) to ensure most reliable experience. You can follow steps mentioned in detail [here](https://learn.microsoft.com/en-us/azure/azure-monitor/app/convert-classic-resource#migrate-your-resource) for migration using any one of the below listed approaches.
```
Comment (asimks) [inline FAQs/README.md:120]: migrate **to** classic Application Insights - > migrate classic Application Insights

File: FAQs/README.md [hunk 5]
```diff
@@ -114,6 +116,25 @@ Using the native enterprise directory for authentication ensures that there is a
...
+<br/>
+
+> [Azure PowerShell](https://learn.microsoft.com/en-us/azure/azure-monitor/app/convert-classic-resource#azure-powershell)
+<br/>
+
+You can use either Workspace in AzTS host Resource group (AzSK-AzTS-LAWorkspace-xxxxx) or any other Workspace (new or existing). This migration will not impact AzTS scan in any way.
+
+We have also updated our deployment package to have this change incorporated. Any new setups/ rerun of installtion script using latest deployment package will result in creation or updation of Application Insights to Workspace-based mode. 
+
+As per latest communication from Azure, for Application Insights not migrated to Workspace-based by retirement date, ingestion will continue until it is auto-migrated.AzTS scan is expected to be non impacted in such cases.  However, for reliable experience, it is recommended to switch to Workspace-based mode by 29th February, 2024.
```
Comment (asimks) [inline FAQs/README.md:135]: AzTS scan is expected to be non impacted in such cases - > AzTS scan will not be impacted due to this.

File: FAQs/README.md [hunk 6]
```diff
@@ -114,6 +116,25 @@ Using the native enterprise directory for authentication ensures that there is a
...
+<br/>
+
+> [Azure PowerShell](https://learn.microsoft.com/en-us/azure/azure-monitor/app/convert-classic-resource#azure-powershell)
+<br/>
+
+You can use either Workspace in AzTS host Resource group (AzSK-AzTS-LAWorkspace-xxxxx) or any other Workspace (new or existing). This migration will not impact AzTS scan in any way.
+
+We have also updated our deployment package to have this change incorporated. Any new setups/ rerun of installtion script using latest deployment package will result in creation or updation of Application Insights to Workspace-based mode. 
+
+As per latest communication from Azure, for Application Insights not migrated to Workspace-based by retirement date, ingestion will continue until it is auto-migrated.AzTS scan is will not be impacted due to this.  However, for reliable experience, it is recommended to switch to Workspace-based mode by 29th February, 2024.
```
Comment (asimks) [inline FAQs/README.md:135]: AzTS scan will not be impacted - remove is

Comment (asimks) [inline FAQs/README.md]: Workspace-based - > Workspace-based mode

**Review by asimks — APPROVED** [review]

**Review by jaiswalricha — APPROVED** [review]

**Pull request merged**